Assets/Scripts/RCON/RCONManager.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Rcon;
+using Rcon.Events;
+using SanAndreasUnity.Utilities;
+using System;
+using System.Collections.Concurrent;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace SanAndreasUnity.RCON
+{
+    public class RCONManager : MonoBehaviour
+    {
+        // Todo : set these from config
+        private static String password;
+        private static int portNumber;
+
+        // Objects used to pass commands and responses between threads
+        private static BlockingCollection<String> mainToSec = new BlockingCollection<String>(1);
+        private static BlockingCollection<String> secToMain = new BlockingCollection<String>(1);
```

Comment:
nice, I like it

also good idea to use `boundedCapacity` as argument for constructor, this way only 1 element can be in a collection